Apollo/Binary/Common.cs

```diff
@@ -26,6 +26,7 @@ public static class Common {
             typeof(Fade),
             typeof(Flip),
             typeof(Hold),
+            typeof(Loop),
```

Comment:
Do not add it in the middle! This will misalign the IDs because they're derived from array index. Move it to the end of the array.